dev/vcpkg/ports/aws-sdk-cpp/vcpkg.json

```diff
@@ -1,8 +1,8 @@
 {
   "$note": "Automatically generated by generateFeatures.ps1",
   "name": "aws-sdk-cpp",
-  "version": "1.11.169",
-  "port-version": 2,
+  "version": "1.11.285",
```

Comment:
Do you know the reason why the version was upgraded to 1.11.285 here but in Velox it was 1.11.321? https://github.com/facebookincubator/velox/pull/9756

Reply:
Seems like they don't have 1.11.321 in ports. https://vcpkg.link/ports/aws-sdk-cpp/versions The nearest version is 1.11.321

Reply:
@marin-ma, yes, vcpkg skips the support for 1.11.321. If there is no reason that Velox must use 1.11.321, maybe, we can upgrade it to a newer version, e.g., 1.11.474, for both Velox and Gluten vcpkg.

Reply:
@marin-ma, it was just upgraded to the latest release (it's 1.11.321 at that time point) in Velox. But I guess we were not aware that vcpkg doesn't support this version, so use a lower version (1.11.285) in Gluten instead. So it's ok to use 1.11.285 or some other higher version at both sides. Maybe, it's better to change Gluten & velox to use a same higher version.

Reply:
https://github.com/apache/incubator-gluten/pull/9169